python/ray/data/read_api.py

```diff
@@ -2282,21 +2283,28 @@ def from_tf(
 @PublicAPI
 def from_torch(
     dataset: "torch.utils.data.Dataset",
+    parallelism: int = -1,
+    shuffle: bool = False,
+    batch_size: int = 32,
 ) -> MaterializedDataset:
     """Create a :class:`~ray.data.Dataset` from a
     `Torch Dataset <https://pytorch.org/docs/stable/data.html#torch.utils.data.Dataset/>`_.
 
-    This function is inefficient. Use it to read small datasets or prototype.
-
     .. warning::
         If your dataset is large, this function may execute slowly or raise an
         out-of-memory error. To avoid issues, read the underyling data with a function
         like :meth:`~ray.data.read_images`.
 
     .. note::
-        This function isn't parallelized. It loads the entire dataset into the head
+        This function uses a streaming implementation. The dataset does not need to fit completely into memory.
+
+    .. note::
+        For iterable-style datasets, this function is not parallelized. It loads the entire dataset into the head
```

Comment:
for iterable datasets we should still do it in a streaming fashion right? it just won't be parallelized.

Reply:
Yup, it should still be streamed

Reply:
The second sentence is not correct then?

Reply:
Yeah that's true, let me remove that